requirements.txt

```diff
@@ -14,3 +14,5 @@ tensorflow-metal; sys_platform == 'darwin' and 'ARM' in platform_version # Mac M
 transformers
 datasets
 torch
+uvicorn
+FastAPI
```

Comment:
Can you please already add 'google-cloud-storage' to the requirements? I have seen on Quentin's branch that he added it. I think we could get conflicts when merging the branches.

Reply:
done. Thank fou for the reminder.